### PR TITLE
Spellchecked against SC2039, still work to be done

### PR DESCRIPTION
**Pull request by Sim4n6**

### CODE REVIEW

File: tecmint_monitor.sh [hunk 1]
```diff
@@ -9,17 +9,17 @@
...
 
 # unset any variable which system may be using
-unset tecreset os architecture kernelrelease internalip externalip nameserver loadaverage
+unset os architecture kernelrelease internalip externalip nameserver loadaverage
 
 #
 # Check for CURL availiability, a dependency of this script
 #
-type -P curl > /dev/null || ( echo "CURL not availiable or not installed, fix prior running"; exit 1 )
+command -v curl > /dev/null || ( echo "CURL not availiable or not installed, fix prior running"; exit 1 )
```
Comment (Sim4n6) [inline tecmint_monitor.sh:17]: type is not POSIX compatible

File: tecmint_monitor.sh [hunk 2]
```diff
@@ -54,21 +54,17 @@ fi
 #
 # Show version info
 #
-if [[ ! -z "$vopt" ]]; then
-	echo -e "tecmint_monitor version 0.1.2\nDesigned by Tecmint.com\nReleased Under Apache 2.0 License"
+if [ ! -z "$vopt" ]; then
+	printf "tecmint_monitor version 0.1.2\nDesigned by Tecmint.com\nReleased Under Apache 2.0 License\n"
```
Comment (Sim4n6) [inline tecmint_monitor.sh:58]: printf is POSIX compatible, rather than echo with option ...

File: tecmint_monitor.sh [hunk 3]
```diff
@@ -44,8 +44,8 @@ if [ ! -z "$iopt" ]; then
 	ok_msg="Congratulations! Script Installed, now run monitor Command"
 	wd=$(pwd)
 	basename "$(test -L "$0" && readlink "$0" || echo "$0")" > /tmp/scriptname
-	scriptname=$(echo -e -n "$wd"/ && cat /tmp/scriptname)
-	su -c "cp $scriptname /usr/bin/monitor" root && echo "${ok_message}" || echo "${fail_msg}"
+	scriptname=$(printf "%s" "$wd"/ && cat /tmp/scriptname)
+	su -c "cp $scriptname /usr/bin/monitor" root && echo "${ok_msg}" || echo "${fail_msg}"
```
Comment (Sim4n6) [inline tecmint_monitor.sh:48]: small mistake

File: tecmint_monitor.sh [hunk 4]
```diff
@@ -44,8 +44,8 @@ if [ ! -z "$iopt" ]; then
 	ok_msg="Congratulations! Script Installed, now run monitor Command"
 	wd=$(pwd)
 	basename "$(test -L "$0" && readlink "$0" || echo "$0")" > /tmp/scriptname
-	scriptname=$(echo -e -n "$wd"/ && cat /tmp/scriptname)
-	su -c "cp $scriptname /usr/bin/monitor" root && echo "${ok_message}" || echo "${fail_msg}"
+	scriptname=$(printf "%s" "$wd"/ && cat /tmp/scriptname)
```
Comment (Sim4n6) [inline tecmint_monitor.sh:47]: printf is POSIX compatible

Comment (atarallo) [reply tecmint_monitor.sh:47]: At least me I'm more used to output with echo, I learned first from the book "The UNIX Programming Environment" by Kernighan & Pike. 

Good job!!

**Pull request merged**